src/pydata_sphinx_theme/theme/pydata_sphinx_theme/components/theme-version.html

```diff
@@ -0,0 +1,7 @@
+<p class="theme-version">
+    {{ _("Based on") }}
+    <a href="https://pydata-sphinx-theme.readthedocs.io/en/stable/index.html">
+        Pydata-sphinx-theme
```

Comment:
```suggestion
        PyData Sphinx Theme
```

I think that matches the PyData brand a bit cleaner